snooty.toml

```diff
@@ -32,7 +32,7 @@ toc_landing_pages = [
 
 [constants]
 download-page = "`downloads page <https://www.mongodb.com/try/download/compass>`__"
-current-version = "1.43.5"
+current-version = "1.44.5"
```

Comment:
[praise] thanks for updating this! I feel like a lot of times we forget to do this 😅